c/src/lib/libbsp/arm/shared/stm32fxxxx/bootinit/hal-bootinit.c

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file hal-bootinit.c
+ *
+ * @ingroup boot
+ *
+ * @brief Triggers the buit-in bootloader. Strictly processor specific.
+ *
+ */
+
+/*
+ * Copyright (c) 2015 Vecna Technologies, Inc.
+ *
+ * The license and distribution terms for this file may be
+ * found in the file LICENSE in this distribution or at
+ * http://www.rtems.com/license/LICENSE.
+ */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <hal-utils.h>
+#include <stm32f-processor-specific.h>
+#include <bspopts.h>
+
+#include stm_processor_header(TARGET_STM_PROCESSOR_PREFIX)
+#include stm_header(TARGET_STM_PROCESSOR_PREFIX, rcc)
+
+void BootloaderInit(
```

Comment:
This isn't following naming conventions. Should be stm32_bootloader_init I believe